sql/ddl_api.sql

```diff
@@ -166,7 +222,7 @@ CREATE OR REPLACE FUNCTION @extschema@.delete_data_node(
     if_exists              BOOLEAN = FALSE,
     force                  BOOLEAN = FALSE,
     repartition            BOOLEAN = TRUE,
-	drop_database          BOOLEAN = FALSE
```

Comment:
Nit: The indention of the other parameters is done with four spaces. For `drop_database`, eight spaces are used.

Reply:
It was actually a tab, so replaced that with (four) spaces to match the other parameters.